.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: node_js
+
+directories:
+- node_modules
+notifications:
+email: false
+branches:
+except:
+- "/^v\\d+\\.\\d+\\.\\d+$/"
+script:
+  - npm test
+  
```

Comment:
remove whitespaces on line 12 without deleting that line

Reply:
done!